hack/deploy.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -e
+
+DEFAULT_OPERATOR_IMAGE="quay.io/cincinnati/cincinnati-operator:latest"
+DEFAULT_OPERAND_IMAGE="quay.io/cincinnati/cincinnati:latest"
+OPERAND_IMAGE="quay.io/app-sre/cincinnati:2873c6b"
+
+if ! [ -n "$CUSTOM_OPERAND_IMAGE" ]; then
+	sed -i "s|$DEFAULT_OPERAND_IMAGE|$OPERAND_IMAGE|" deploy/operator.yaml
+else
+	sed -i "s|$DEFAULT_OPERAND_IMAGE|$CUSTOM_OPERAND_IMAGE|" deploy/operator.yaml
+fi
```

Comment:
You could do something like:
```OPERAND_IMAGE="{$OPERAND_IMAGE:-${DEFAULT_OPERAND_IMAGE}}"```

and move the sed statements outside of each if-else, as they are applicable in each case.
```
sed -i "s|$DEFAULT_OPERAND_IMAGE|$OPERAND_IMAGE|" deploy/operator.yaml
sed -i "s|$DEFAULT_OPERATOR_IMAGE|$OPERATOR_IMAGE|" deploy/operator.yaml
sed -i "s|your-registry/your-repo/your-init-container|$GRAPH_DATA_IMAGE|" deploy/crds/cincinnati.openshift.io_v1beta1_cincinnati_cr.yaml
```